app/keepers.go

```diff
@@ -123,6 +123,8 @@ import (
 	bech32ibckeeper "github.com/osmosis-labs/bech32-ibc/x/bech32ibc/keeper"
 	bech32ibctypes "github.com/osmosis-labs/bech32-ibc/x/bech32ibc/types"
 	bech32ics20keeper "github.com/osmosis-labs/bech32-ibc/x/bech32ics20/keeper"
+
+	owasm "github.com/osmosis-labs/osmosis/v7/app/wasm"
```

Comment:
why `osmosis/v7`?
I don't see this directory. Is this pinning to a commit?

Reply:
This is what the IDE (Intellij) suggests. Thought it was correct. Doesn't this depends on go.mod or so? Will take a look at the other packages, to see what they import.